### PR TITLE
cam 1.3: Create supplemental group for Restic on NFS

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=1881456

CP for 4.5, 4.6

### CODE REVIEW

**Comment by XinRedhat** [comment]
LGTM

File: modules/migration-restic-errors.adoc [hunk 1]
```diff
@@ -115,3 +115,32 @@ The output identifies the Restic Pod that logged the errors.
...
+[source,terminal]
+----
+backup=openshift-migration/<backup_id> controller=pod-volume-backup error="fork/exec /usr/bin/restic: permission denied" error.file="/go/src/github.com/vmware-tanzu/velero/pkg/controller/pod_volume_backup_controller.go:280" error.function="github.com/vmware-tanzu/velero/pkg/controller.(*podVolumeBackupController).processBackup" logSource="pkg/controller/pod_volume_backup_controller.go:280" name=<backup_id> namespace=openshift-migration
+----
+
+You can resolve this issue by creating a supplemental group for Restic and adding the group ID to the Migration Controller.
+
+.Procedure
+
+. Create a xref:../../storage/persistent_storage/persistent-storage-nfs.adoc#group-ids[supplemental group] using a common group.
```
Comment (adellape) [inline modules/migration-restic-errors.adoc:134]: Unfortunately, we can't use `xref`s from within a module. You can move the link to an "Additional resources" list after this module in the assembly, however.

File: modules/migration-restic-errors.adoc [hunk 2]
```diff
@@ -9,7 +9,7 @@
 [id='restic-timeout-error-velero-log_{context}']
 == Restic timeout error message in the Velero Pod log
 
-If a migration fails because Restic times out, the following error appears in the Velero Pod log.
+If a migration fails because Restic times out, the following error is displayed in the Velero Pod log.
```
Comment (adellape) [inline modules/migration-restic-errors.adoc:12]: Can all of these changes of "appears" to "is displayed" stay in active voice? E.g., "the following error displays"?

**Review by adellape — COMMENTED** [review]
A few comments.

File: modules/migration-restic-errors.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-restic-errors.adoc:134]: Oops! Will fix

File: modules/migration-restic-errors.adoc (hunk 2 above)
Comment (apinnick) [reply modules/migration-restic-errors.adoc:12]: No. IBM is quite clear about that. "Displays" is not an intransitive verb.

File: modules/migration-restic-errors.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-restic-errors.adoc:134]: Actually, I'll just take the xref out. If someone doesn't know how to create a supplemental group, they have no business doing a migration. :-)

**Comment by apinnick** [comment]
@adellape Changes made. Ready to merge

File: modules/migration-restic-errors.adoc (hunk 2 above)
Comment (adellape) [reply modules/migration-restic-errors.adoc:12]: Ah ha, got it mixed up, I see it now. Thanks!

**Comment by apinnick** [comment]
@adellape Can you merge this? CAM 1.3 is supposed to be GA today.

**Comment by apinnick** [comment]
@adellape Please merge

File: modules/migration-restic-errors.adoc [hunk 3]
```diff
@@ -115,3 +115,32 @@ The output identifies the Restic Pod that logged the errors.
 ----
 $ oc logs -f <restic-nr2v5>
 ----
+
+ifeval::["{mtc-version}" < "1.3"]
+== Restic permission error for NFS storage
```
Comment (kalexand-rh) [inline modules/migration-restic-errors.adoc:120]: You need an explicit anchor for the heading.

File: modules/migration-restic-errors.adoc [hunk 4]
```diff
@@ -9,7 +9,7 @@
 [id='restic-timeout-error-velero-log_{context}']
```
Comment (kalexand-rh) [inline modules/migration-restic-errors.adoc:9]: This module needs a different title and an intro statement. It looks like it's a reference for resolving error messages.

Comment (apinnick) [reply modules/migration-restic-errors.adoc]: Done

**Pull request merged**